selectors.yml

```diff
@@ -107,18 +146,30 @@ selectors:
   421614:
     selector: 3478487238524512106
     name: "ethereum-testnet-sepolia-arbitrum-1"
+  432201:
+    selector: 1458281248224512906 
+    name: "dexalot-subnet-testnet"
```

Comment:
subnet-testnet? :thinking:

Reply:
yea the subnet has a testnet. https://subnets-test.avax.network/dexalot

Reply:
Interesting. Is this an avalanche subnet? I don't think we have prior art for this kind of stuff on naming proposal; should we come up with something?
How about `avalanche-subnet-dexalot-testnet`? i.e. the testnet of the dexalot subnet attached to avalanche (like testnets and rollups)

Reply:
Why not follow RDD naming convention instead of creating a new one?

Reply:
@makramkd that's the [whole idea of this repo](https://smartcontract-it.atlassian.net/wiki/spaces/DEPLOY/pages/599491447/Network+naming+proposal). But there's some room for decision still on that document (e.g. it doesn't anticipate these subnets), and as such we need to find the schema which best matches the "spirit" of the whole proposal.

Reply:
FWIW I think avalanche-subnet-dexalot-testnet matches RDD naming convention, but would be good to get engop's review on this.

Reply:
added!